pkg/session/bootstrap.go

```diff
@@ -120,6 +120,7 @@ const (
 		Password_expired		ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Password_last_changed	TIMESTAMP DEFAULT CURRENT_TIMESTAMP(),
 		Password_lifetime		SMALLINT UNSIGNED DEFAULT NULL,
+		Max_user_connections 	SMALLINT UNSIGNED DEFAULT 0,
```

Comment:
I see in MySQL8 it is int unsigned rather than smallint unsigned.

```
  `max_connections` int unsigned NOT NULL DEFAULT '0',
```

Reply:
modify it.